### PR TITLE
REST: Add OpenAPI json

### PR DESCRIPTION
**Pull request by ashleyvega**
This PR:
- moves proto files from the `grpc` dir back to the top level dir (to avoid `../elsewhere/` in app arguments)
- adds generation of OpenAPI (swagger) json (see https://www.openapis.org/)

### CODE REVIEW

File: rest/grpc-rest-bindings.yml [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    - selector: api.v1.TradingDataService.MarketAccounts
+      get: '/markets/{market_id}/accounts'
+
+    # assets
+    - selector: api.v1.TradingDataService.Assets
+      get: '/assets'
+    - selector: api.v1.TradingDataService.AssetByID
+      get: '/assets/{id}'
+    - selector: api.v1.TradingDataService.FeeInfrastructureAccounts
+      get: '/assets/{asset}/infrastructureFeeAccount'
```
Comment (fkondej) [inline rest/grpc-rest-bindings.yml:40]: Is the `{asset}` correct in here?

**Review by fkondej — CHANGES_REQUESTED** [review]
Other LGTM.

File: rest/grpc-rest-bindings.yml (hunk 1 above)
Comment (ashleyvega) [reply rest/grpc-rest-bindings.yml:40]: Yes, it's correct.

- `AssetByIDRequest` has `string id`: https://github.com/vegaprotocol/vega/blob/e5bc06543b7b1ae87d4fb4888168d04586bd7b6f/proto/api/trading.proto#L380-L384
- `FeeInfrastructureAccounts` has `string asset`: https://github.com/vegaprotocol/vega/blob/e5bc06543b7b1ae87d4fb4888168d04586bd7b6f/proto/api/trading.proto#L984-L990

File: .merge.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import json
+import sys
+
+
+# https://stackoverflow.com/questions/7204805
+def merge(a, b):
```
[the remark this answers is not in the document]
Comment (ashleyvega) [reply .merge.py]: Done in 81ccca1.

**Review by fkondej — APPROVED** [review]
LGTM

**Pull request merged**